jacodb-ets/src/main/kotlin/org/jacodb/ets/model/EtsMethod.kt

```diff
@@ -26,10 +26,23 @@ interface EtsMethod : CommonMethod {
     val signature: EtsMethodSignature
     val localsCount: Int
     val cfg: EtsCfg
+    val modifiers: List<String>
 
     val enclosingClass: EtsClassSignature
         get() = signature.enclosingClass
 
+    val isStatic: Boolean
+        get() = modifiers.contains("StaticKeyword")
+
+    val isPrivate: Boolean
+        get() = modifiers.contains("PrivateKeyword")
+
+    val isPublic: Boolean
+        get() = !isPrivate
```

Comment:
Let's make it consistent: there also exist an explicit `PublicKeyword` modifier.